metric_tank/aggmetrics.go

```diff
@@ -48,10 +48,13 @@ func NewAggMetrics(store Store, chunkSpan, numChunks, chunkMaxStale, metricMaxSt
 // periodically scan chunks and close any that have not received data in a while
 // TODO instrument occurences and duration of GC
 func (ms *AggMetrics) GC() {
-	ticker := time.Tick(ms.gcInterval)
-	for now := range ticker {
+	for {
+		unix := time.Duration(time.Now().UnixNano())
```

Comment:
give everything else here is working with seconds, shouldn't this be Unix() and not UnixNano()?